ibazel/main.go

```diff
@@ -24,7 +24,20 @@ import (
 	"github.com/fsnotify/fsnotify"
 )
 
-var b *bazel.Bazel
+type State string
+
+const (
+	DEBOUNCE_QUERY State = "DEBOUNCE_QUERY"
+	QUERY          State = "QUERY"
+	WAIT           State = "WAIT"
+	DEBOUNCE_RUN   State = "DEBOUNCE_RUN"
+	RUN            State = "RUN"
+	QUIT           State = "QUIT"
+)
+
+const debounceDuration = 100 * time.Millisecond
+const sourceQuery = "kind('source file', deps('%s'))"
+const buildQuery = `let d = deps(set(%s)) in buildfiles($d)`
```

Comment:
I think you can just write `buildfiles(deps(set(%s)))`, no need for the variable.